CotEditor/en-GB.lproj/Acknowledgments.html

```diff
@@ -0,0 +1,216 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8"/>
+    <title>Acknowledgments</title>
+    <link rel="stylesheet" href="../Base.lproj/Acknowledgments.css"/>
+</head>
+
+<body>
+<h1>Acknowledgments</h1>
+
+<p>CotEditor is developed using the following awesome technologies. We are deeply grateful for those who let us use their valuable programs/information.</p>
+
+
+<section id="Libraries">
+<section class="Sparkle">
+<h3>Sparkle <span>(only on non-AppStore ver.)</span></h3>
+<p><a href="https://sparkle-project.org">https://sparkle-project.org</a></p>
+<p>Thanks to Sparkle, the software update framework, by Andy Matuschak et al. Sparkle is distributed under the <em>MIT licence</em>.</p>
+
+<details>
+<summary>MIT licence</summary>
+<blockquote><pre>
+© 2006-2013 Andy Matuschak
+© 2009-2013 Elgato Systems GmbH.
+© 2011-2014 Kornel Lesiński
+© 2015-2017 Mayur Pawashe
+© 2014 C.W. Betts
+© 2014 Petroules Corporation
+© 2014 Big Nerd Ranch
+All rights reserved.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the &quot;Software&quot;), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicence, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED &quot;AS IS&quot;, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+=================
+EXTERNAL LICENCES
```

Comment:
Please revert `LICENCES` to `LICENSES`, because this line is a part of the license paragraphs. It should be kept as-is.

Reply:
Fixed in 39de168